### PR TITLE
Remove pool and or

### PR DESCRIPTION
**Pull request by Colstuwjx**
This PR carried remove pool thing, and to prove that remove pool with _or operator works well.

### CODE REVIEW

File: builder/builder.go [hunk 1]
```diff
@@ -191,19 +187,6 @@ func BuildReplaceInsert(table string, data []map[string]interface{}) (string, []
...
-		},
-	}
-)
-
-func getCpPool() ([]Comparable, func()) {
-	obj := cpPool.Get().([]Comparable)
-	return obj[:0], func() { cpPool.Put(obj) }
-}
-
 func emptyFunc() {}
```
Comment (Colstuwjx) [inline builder/builder.go:190]: this line also need to be removed?

**Comment by Colstuwjx** [comment]
This PR is ONLY for demo, **DO NOT MERGE**.

File: builder/builder.go (hunk 1 above)
Comment (caibirdme) [reply builder/builder.go:190]: Yes, emptyFunc is meant for satisfy the function signature, and it's useless now

**Comment by caibirdme** [comment]
I think this change is great, but I will remove the sync.Pool first, and then merge your pr.
And I've benchmarked the change, the performance isn't affected actually

**Comment by Colstuwjx** [comment]
Sure, pls do!

**Comment by caibirdme** [comment]
> Sure, pls do!

I've already merged that pr, pls update your own, and make sure it can pass all the test cases.

**Comment by Colstuwjx** [comment]
I'll update the original one, this PR is just for demo usage :)

**Comment by Colstuwjx** [comment]
Close since #77 complete!

**Pull request closed without merging**